src/app/policies/components/PolicyList.vue

```diff
@@ -94,7 +94,7 @@
             :empty-state-cta-text="t('common.documentation')"
             :headers="[
               { label: 'Name', key: 'name' },
-              ...(props.currentPolicyType.isTargetRefBased ? [{ label: 'Zone', key: 'zone' }] : []),
+              ...(props.currentPolicyType.isTargetRefBased || props.currentPolicyType.name === 'MeshGateway' ? [{ label: 'Zone', key: 'zone' }] : []),
```

Comment:
**Comment**: This is technically a small bug fix. We should show the Zone column for MeshGateway. However, this will soon go away again because MeshGateways won’t be shown under policies in the future.